### PR TITLE
Fixing copy button on invitation flow

### PR DESCRIPTION
**Pull request by jespino**
Fixing copy button on invitation flow

### CODE REVIEW

File: components/invitation_modal/invitation_modal_members_step.jsx [hunk 1]
```diff
@@ -30,11 +31,30 @@ export default class InvitationModalMembersStep extends React.Component {
...
         };
     }
 
+    copyLink = () => {
+        const input = this.inviteLinkRef.current;
+        input.focus();
+        input.setSelectionRange(0, input.value.length);
+
+        try {
+            if (document.execCommand('copy')) {
```
Comment (reflog) [inline components/invitation_modal/invitation_modal_members_step.jsx]: Nit 0/1:
```
this.setState({copiedLink: document.execCommand('copy')});
```

File: components/invitation_modal/invitation_modal_members_step.jsx [hunk 2]
```diff
@@ -68,6 +88,18 @@ export default class InvitationModalMembersStep extends React.Component {
 
     render() {
         const inviteUrl = getSiteURL() + '/signup_user_complete/?id=' + this.props.inviteId;
+        let copyLinkConfirm = null;
+        if (this.state.copiedLink) {
+            copyLinkConfirm = (
+                <p className='alert alert-success alert--confirm'>
+                    <SuccessIcon/>
+                    <FormattedMessage
+                        id='get_link.clipboard'
```
Comment (reflog) [inline components/invitation_modal/invitation_modal_members_step.jsx]: no changes to i18n ?

File: components/invitation_modal/invitation_modal_members_step.jsx [hunk 3]
```diff
@@ -68,6 +88,18 @@ export default class InvitationModalMembersStep extends React.Component {
...
+                        defaultMessage=' Link copied'
```
Comment (reflog) [inline components/invitation_modal/invitation_modal_members_step.jsx]: Leading space intended?

File: components/invitation_modal/invitation_modal_members_step.jsx (hunk 2 above)
Comment (jespino) [reply components/invitation_modal/invitation_modal_members_step.jsx]: This comes from other part of the code, concretely from `components/get_link_modal.jsx`

File: components/invitation_modal/invitation_modal_members_step.jsx (hunk 3 above)
Comment (jespino) [reply components/invitation_modal/invitation_modal_members_step.jsx]: This comes from other part of the code, concretely from `components/get_link_modal.jsx`

File: components/invitation_modal/invitation_modal_members_step.jsx (hunk 1 above)
Comment (jespino) [reply components/invitation_modal/invitation_modal_members_step.jsx]: Done! :)

**Pull request merged**